analyzer/analyzer.go

```diff
@@ -176,6 +176,7 @@ func (ac Config) analyzeLayer(ctx context.Context, dig digest.Digest) (digest.Di
 	}
 
 	layerInfo := types.LayerInfo{
+		ID:            dig,
```

Comment:
I think we don't need to store `ID` here because a cache key already has that information. We can reduce the cache size.

Reply:
Good point. I will remove this.